db/migrate/20201228185540_create_recipes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecipes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :recipes do |t|
+      t.string :image
+      t.string :title
+      t.text :description
+      t.string :instructions
+      t.integer :charity_id
```

Comment:
I forget how we decided to handle the charities. Will we be making a Propublica request each time to get charities or will we store the charities?